ros2doctor/ros2doctor/api/format.py

```diff
@@ -12,9 +12,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import sys
 
-def print_term(k, v):
-    """Print term only if it exists."""
-    if v:
-        # TODO(clairewang): 20 padding needs to be dynamically set
-        print('{:20}: {}'.format(k, v))
+
+def format_print(report):
+    """Format print report content."""
+    # temp fix for missing ifcfg
+    if report is None:
+        sys.stderr.write('No report found. Skip print...')
+        return
+
+    print('\n', report.name)
```

Comment:
The comma adds a leading space. Is that intentional?

```
 ROS 2 INFORMATION
distribution name      : eloquent
```

Reply:
It makes the header stands out more so I left it in. I can change it if it seems weird.

Reply:
No need to change, just checking it wasn't accidental